package.json

```diff
@@ -3,9 +3,10 @@
   "scripts": {
     "build-css": "node-sass --source-map true --include-path ./node_modules/uswds/src/stylesheets tock/tock/static/sass -o tock/tock/static/css/dist",
     "watch-css": "npm run build-css -- --watch",
-    "prestart": "npm install --quiet",
+    "prestart": "CYPRESS_INSTALL_BINARY=0 npm install --quiet",
     "start": "npm run build-css && npm run watch-css",
-    "predeploy": "npm install --silent && npm run build-css"
+    "predeploy": "CYPRESS_INSTALL_BINARY=0 npm install --silent && npm run build-css",
+    "cypress:open": "cypress open"
```

Comment:
Hey @davemcorwin can we stick to the `-` dashed convention we have for `npm run scripts` rather than the `:` colon convention?

Reply:
yep!